ajax/libs/bitset.js/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "bitset.js",
+  "filename": "bitset.min.js",
+  "version": "4.0.0",
+  "homepage": "http://www.xarg.org/2014/03/javascript-bit-array/",
+  "description": "A performance optimized infinite bit vector library",
+  "keywords": [
+    "bit",
+    "set",
+    "twiddle",
+    "bit array",
+    "binary",
+    "bit vector",
+    "bitset",
+    "bitmap",
+    "bitstring",
+    "bitwise"
+  ],
+  "author": {
+    "name": "Robert Eisele <robert@xarg.org>",
+    "url": "http://www.xarg.org/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/infusion/bitset.js.git"
+  },
+  "npmName": "bitset.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
hi @mercyg there is no `dist` folder in 4.0.0 in npm package, please just leave a blank as basepath, and modify filemap to `bitset*`.